tests/core/test_exp_manager.py

```diff
@@ -195,6 +195,32 @@ def test_trainer_loggers(self, tmp_path):
         )
         assert isinstance(test_trainer.logger, pl.loggers.WandbLogger)
 
+        test_trainer = pl.Trainer(accelerator='cpu', logger=False)
```

Comment:
Create a separate test, and to the import check via try scratch to run the test only if Neptune is manually installed.

Reply:
Could you elaborate a bit? Do you mean because we are removing the default dependency, you want me to check for availability of neptune and then only run these tests for neptune? Is my understanding correct?

Reply:
Hi @titu1994  I have used conditional skip to skip the test if neptune cannot be imported. Hope this will help to achieve your suggestion.